template/src/themes/metrics.ts

```diff
@@ -6,15 +6,15 @@ const DESIGN_WIDTH = 375
 const DESIGN_HEIGHT = 812
 const {width, height} = Dimensions.get('window')
 
-function responsiveWidth(value: number = 0): number {
+function responsiveWidth(value = 0): number {
   return (width * value) / DESIGN_WIDTH
 }
 
-function responsiveHeight(value: number = 0): number {
+function responsiveHeight(value = 0): number {
   return (height * value) / DESIGN_HEIGHT
 }
 
-function responsiveFont(value: number = 0): number {
+function responsiveFont(value = 0): number {
```

Comment:
Because we already given them a default value, we don't need to define type for them. This is also an error from ESLint
<img width="400" alt="Screen Shot 2023-02-27 at 09 45 40" src="https://user-images.githubusercontent.com/114563576/221461381-4e4912dc-a130-4fdd-a566-daac1d29627d.png">